### PR TITLE
Clamp the height of description text for property selectors

### LINKED ISSUE #83495: Editor property selector does not clamp description
### Godot version

4.2 beta1

### System information

Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1060 (NVIDIA; 30.0.15.1403) - Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 Threads)

### Issue description

https://github.com/godotengine/godot/assets/2223172/17fe3cb5-c852-4094-b1de-cfad9da6f04a

The description should be scrollable probably.

### Steps to reproduce

1. Add AnimationPlayer and animation
2. Click Add Track
3. Select some node and click a property with a long description

### Minimal reproduction project

N/A

### PR DESCRIPTION
**Pull request by quirkylemon**
Fixes #83495 
### New
Screenshot from 2023-10-21 13-33-58
### Old 
Screenshot from 2023-10-21 13-37-08

### CODE REVIEW

**Review by SaracenOne — APPROVED** [review]
Yeah, tested and this seems good. Much better UX. 👍

File: editor/property_selector.cpp [hunk 1]
```diff
@@ -569,5 +569,7 @@ PropertySelector::PropertySelector() {
 
 	help_bit = memnew(EditorHelpBit);
 	vbc->add_margin_child(TTR("Description:"), help_bit);
+	help_bit->get_rich_text()->set_fit_content(false);
+	help_bit->get_rich_text()->set_custom_minimum_size(Size2(help_bit->get_rich_text()->get_minimum_size().x, 135));
```
Comment (Calinou) [inline editor/property_selector.cpp:573]: Scale to hiDPI displays:
```suggestion
help_bit->get_rich_text()->set_custom_minimum_size(Size2(help_bit->get_rich_text()->get_minimum_size().x, 135 * EDSCALE));
```

Comment (quirkylemon) [reply editor/property_selector.cpp:573]: I am new to github (this is my first PR ever). do I need to do anything with this?

Comment (Calinou) [reply editor/property_selector.cpp:573]: Please make the change to your local copy, stage your changes, then force-push an amended commit as described in https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#the-interactive-rebase.

Comment (quirkylemon) [reply editor/property_selector.cpp]: I made the changes

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**